src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  padding-inline: 16px;
+  display: flex;
+  justify-content: space-between;
+  position: absolute;
+  top: 276px;
+  width: 166px;
+
+  &__stars-box {
+    display: flex;
+    width: 96px;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
```

Comment:
Verify that the image `../images/star.svg` exists at the specified path to ensure the background image is displayed correctly.